src/helpers/generateConfig.js

```diff
@@ -0,0 +1,28 @@
+
+import interopRequireDefault from './interopRequireDefault';
+import getModuleFromPath from './getModuleFromPath';
+
+function getSyncModule(config, syncScriptPath) {
+  const {
+    projectRoot,
+  } = config;
+
+  return getModuleFromPath(syncScriptPath, {
+    projectRoot: projectRoot || process.cwd(),
+    requireInteropWrapper: interopRequireDefault,
+  });
+}
+
+export default function generateConfig(config) {
+  const {
+    sync,
+  } = config;
+
+  return new Promise((resolve) => {
```

Comment:
should this use https://www.npmjs.com/package/promise.try ?

Reply:
^ Seems like it would be a good idea